src/components/postgresConnect.ts

```diff
@@ -692,16 +692,18 @@ class PostgresDB {
       WHERE id = $1
       `
     const text2 = `
-      SELECT name as "orgName"
-      FROM organisation
-      WHERE id = $1
+    SELECT distinct("orgName"), "permissionName"
+      FROM permissions_all
+      WHERE ("userId" = $1 OR "userId" IS NULL) 
+      AND "orgId" = $2
```

Comment:
I'm getting an error when calling `getApplicationData` endpoint on an application with no orgId.

The problem is here -- you're only returning the permissions info when there is an orgId provided, but it should return it regardless.

Reply:
This query is also called by `getUserInfo`/`getUserOrgData` (in loginHelpers), so I'm pretty sure it'll break that by changing it here too.

Everything you need is already done in the `getUserInfo` function, so you can probably just use that in "getApplicationData" (instead of `getUserData`) and modify the result into the shape you need it to be in "getApplicationData" (it includes all the permission name info too, I'm pretty sure.)

I'm very reluctant to change any of these queries, as they're very fragile and have subtle implications in lots of different places.

Reply:
Okay, have tried my suggestion and it seems to work okay. It manages to get what we need without changing any postGresConnect queries, which I think is important considering how interconnected they are.

Branch is #1445B-move-checks-for-admin-manager-inside-current-user_suggestion

Compare here: https://github.com/openmsupply/conforma-server/compare/%231445B-move-checks-for-admin-manager-inside-current-user...%231445B-move-checks-for-admin-manager-inside-current-user_suggestion

(All the changes in `postgresConnect.ts` are just reverting it back to `develop`)

Reply:
Nice Carl! That looks great to me